collections/_catalog/deployment/2df29b2b-7e5e-4d8b-a5b3-3dea569f4d25.md

```diff
@@ -18,7 +18,7 @@ compatibility:
 patternId: 2df29b2b-7e5e-4d8b-a5b3-3dea569f4d25
 image: /assets/images/patterns/service-mesh.svg
 patternInfo: This design includes a WordPress site and a MySQL database using Minikube. Both applications use PersistentVolumes and PersistentVolumeClaims to store data.
-patternCaveats: Warning: This deployment is not suitable for production use cases, as it uses single instance WordPress and MySQL Pods. Consider using WordPress Helm Chart to deploy WordPress in production.
+patternCaveats: 'Warning: This deployment is not suitable for production use cases, as it uses single instance WordPress and MySQL Pods. Consider using WordPress Helm Chart to deploy WordPress in production.'
```

Comment:
@leecalcote @vaibhav-xt You're right with adding quotes here.
If the string contains any special character including `:` then the YAML string requires quotes. See this post for other special characters in list: https://stackoverflow.com/a/22235064/11966205

Reply:
@Abhishek-kumar09 
I was not answering this because I am not sure about this in `YAML`.

Reply:
I'll suggest that `|` is used.